tests/components/husqvarna_automower/const.py

```diff
@@ -0,0 +1,226 @@
+"""Helpers for testing Husqvarana Automower."""
+
+MWR_ONE_ID = "c7233734-b219-4287-a173-08e3643f89f0"
+MWR_ONE_IDX = 0
+MWR_TWO_ID = "1c7aec7b-06ff-462e-b307-7c6ae4469047"
+MWR_TWO_IDX = 1
+
+
+MOWER_ONE_SESSION_DATA = {
```

Comment:
The large data constants can be stored as JSON files in a `fixtures` directory under this integration tests. We have helpers to load fixture files.

https://github.com/home-assistant/core/blob/4ae5757bc175307545e45519cb64a1d24b9f763f/tests/common.py#L470-L494